AppOne/AppOne/Model/SectionTitles.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  SectionTitles.swift
+//  AppOne
+//
+//  Created by Asapu, Ajaikumar on 21.11.22.
+//
+
+struct SectionTitles : Hashable {
+    let title: String
+
+    func hash(into hasher: inout Hasher) {
+        hasher.combine(title)
+    }
```

Comment:
can be removed

Reply:
don't we need hasher ever here?

Reply:
If you conform to Hashable and all properties conform to Hashabale, then you get the default implementation over all properties for free from the compiler